net/respondd-module-wifi/src/respondd.c

```diff
@@ -0,0 +1,124 @@
+#include <string.h>
+#include <stdio.h>
+#include <json-c/json.h>
+#include <respondd.h>
+
+#include <linux/nl80211.h>
+
+#include "airtime.h"
+#include "clients.h"
+#include "neighbours.h"
+#include "ifaces.h"
+
+static struct json_object *respondd_provider_statistics(void) {
+	struct json_object *result, *wireless, *clients, *interface;
+	struct iface_list *ifaces;
+	int wifi24 = 0, wifi5 = 0, clients_count;
+
+	result = json_object_new_object();
+	if (!result)
+		return NULL;
+
+	wireless = json_object_new_array();
+	if (!wireless) {
+		//TODO why needed? : json_object_put(result);
```

Comment:
there are still some TODO in here. I think this one is relevant. result has been initialized as new json objet and must be freed.

Reply:
Why is there no regular `json_free()` function? It should not be so hacky with a `json_object_put()`,